client/tailscale/localclient.go

```diff
@@ -1019,6 +1019,23 @@ func (lc *LocalClient) DebugDERPRegion(ctx context.Context, regionIDOrCode strin
 	return decodeJSON[*ipnstate.DebugDERPRegionReport](body)
 }
 
+func (lc *LocalClient) StartDebugCapture(ctx context.Context, addr string) error {
+	v := url.Values{"addr": {addr}}
+	body, err := lc.send(ctx, "POST", "/localapi/v0/start-debug-capture?"+v.Encode(), 200, nil)
+	if err != nil {
+		return fmt.Errorf("error %w: %s", err, body)
+	}
+	return nil
+}
+
+func (lc *LocalClient) StopDebugCapture(ctx context.Context) error {
```

Comment:
stop what? there can only be 1 capture globally?

then why doesn't it end when the HTTP request above ends? the one above can stream a pcap?

Reply:
Currently the way it works is you install a hook, which is an atomic func callback, and that gets wired to all sockets spawned from a TCP listener. I'm not crashhot on trying to wire it from the localAPI socket but I guess we could try that.

I lean towards TCP with a default localhost address but the ability to open it up via the command line, so then you can capture on one machine and monitor on another.